pom.xml

```diff
@@ -79,6 +79,7 @@
 		<module>springdoc-openapi-starter-webflux-api</module>
 		<module>springdoc-openapi-starter-webmvc-ui</module>
 		<module>springdoc-openapi-starter-webflux-ui</module>
+		<module>springdoc-openapi-tests</module>
```

Comment:
Hi, @bnasslahsen , please note the new module `springdoc-openapi-tests` here. After adding it, https://github.com/springdoc/springdoc-openapi/commit/3eddb9d88dfde16be3224ab49139b09d2738cf46 and https://github.com/springdoc/springdoc-openapi/commit/72265643923890b312eb05cf42233a5aadc02267 will update the version automatically instead of updating manually.

If we care about the deployment for the module `springdoc-openapi-tests`, don't forget to configure the plugin `maven-deploy-plugin` at `springdoc-openapi-tests/pom.xml`:

```xml
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-deploy-plugin</artifactId>
  <configuration>
    <skip>true</skip>
  </configuration>
</plugin>
```

Reply:
@JohnNiang, 

Thank you for your hint. For `springdoc-openapi-tests` it won't be necessary to keep deploying this module.